Azure/azure-cli
# [AKS] `az aks mesh`: Add `az aks mesh` commands to manage Azure Service Mesh in given cluster

Add az aks mesh commands

az aks mesh enable
az aks mesh disable
az aks mesh enable-ingress-gateway
az aks mesh disable-ingress-gateway
az aks mesh get-revisions
az aks mesh get-upgrades

**Related command**
az aks create --enable-azure-service-mesh

**Description**
Adding Azure Service Mesh commands.

**Testing Guide**
az aks mesh enable --resource-group MyResourceGroup --name MyManagedCluster
az aks mesh disable --resource-group MyResourceGroup --name MyManagedCluster
az aks mesh enable-ingress-gateway --resource-group MyResourceGroup --name MyManagedCluster --ingress-gateway-type Internal
az aks mesh get-revisions --location westus

This checklist is used to make sure that common guidelines for a pull request are followed.
